packages/frontend-web/src/app/platform/websocketService.ts

```diff
@@ -123,25 +123,37 @@ function packArticleData(data: any): IAllArticlesData {
 }
 
 function packArticleUpdate(data: any): IArticleUpdate {
-  const cdata = data.category;
-  let cmodel;
-  let amodel;
-
-  if (cdata) {
-    cdata.id = cdata.id.toString();
-    cmodel = CategoryModel(cdata);
+  let categories;
+  let articles;
+ 
+  
+  if(data.categories) {
```

Comment:
Great call to introduce a new (plural) name for the Lists. This is much more readable than introspecting the type (as I recommended.)